bar-trender-front-end/src/components/FreePieChart.js

```diff
@@ -1,42 +1,91 @@
-import React, {Component} from 'react';
+import React, { useEffect, useState, Component} from 'react';
 import {Bar, Line, Pie} from 'react-chartjs-2';
 
-class BeerPieChart extends Component{
-  constructor(props){
-    super(props);
-    this.state = 
-      this.props.data;
+
+  function BeerPieChart(props) {
     
-  }
+    const [modal1, setModal1] = React.useState(false);
+    const [modal2, setModal2] = React.useState(false);
+    const { element } = props;
 
-  static defaultProps = {
-    displayTitle:true,
-    displayLegend: true,
-    legendPosition:'right',
-    location:'City'
-  }
 
-  render(){
+    
+    const [appState, setAppState] = useState({
+      stats: {},
+    });
+    
+
+ 
+  
+     useEffect(() => {
+      var token = sessionStorage.getItem("token");
+      console.log(token,'token');
+      const apiUrl = "http://develop-backend-sprint-01.herokuapp.com/v1/stats/get";
+      async function loadStats(){
+        await fetch(apiUrl, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+            'token': token,
+          },
+         
+          body: JSON.stringify({ filter : 'Bebidas' })
+        }).then(response => response.json())
+          .then(stats => {
+            setAppState({ stats: stats });
+          });} 
+          loadStats()
+    },[setAppState]);
+  
+
+    var graph2 = {
+      chartData:{
+        labels: [appState.stats.first.name, appState.stats.second.name, appState.stats.third.name, 'Otros'],
+        datasets:[
+          {
+            label:'Busquedas',
+            data:[
+              appState.stats.first.percentage,
+              appState.stats.second.percentage,
+              appState.stats.third.percentage,
+              appState.stats.other.percentage,
+            ],
+            backgroundColor:[
+              'rgba(255, 99, 132, 0.6)',
+              'rgba(54, 162, 235, 0.6)',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7173304)
```suggestion
              "rgba(54, 162, 235, 0.6)",
```